integration-test/ping_test.js

```diff
@@ -12,7 +12,7 @@ describe('Ping', () => {
   let nodeIds = []
 
   before(() => {
-    return loadTestNodeIds().then(res => nodeIds = res)
+    return loadTestNodeIds().then(res => { nodeIds = res })
```

Comment:
Yeah, that makes sense, my brain definitely can't isolate the expression/statement rules from perl vs es6 vs scala so might as well be safe